Code/GraphMol/Chirality.cpp

```diff
@@ -828,29 +828,70 @@ std::pair<bool, bool> assignBondStereoCodes(ROMol &mol, UINT_VECT &ranks) {
               endDir = endAtomNeighbors[1].second;
               endNbrAid = endAtomNeighbors[1].first;
             }
-            dblBond->getStereoAtoms().push_back(begNbrAid);
-            dblBond->getStereoAtoms().push_back(endNbrAid);
-            if (hasExplicitUnknownStereo) {
-              dblBond->setStereo(Bond::STEREOANY);
-              assignedABond = true;
-            } else if (begDir == endDir) {
-              // In findAtomNeighborDirHelper, we've set up the
-              // bond directions here so that they correspond to
-              // having both single bonds START at the double bond.
-              // This means that if the single bonds point in the same
-              // direction, the bond is cis, "Z"
-              dblBond->setStereo(Bond::STEREOZ);
+
+            bool conflictingBegin = false;
+            bool conflictingEnd = false;
+            if (begAtomNeighbors.size() == 2 &&
```

Comment:
This might be more clear as:

```
const bool conflictingBegin = (begAtomNeighbors.size() == 2 && 
           begAtomNeighbors[0].second == begAtomNeighbors[1].second);
const bool conflictingEnd = (endAtomNeighbors.size() == 2 &&
           endAtomNeighbors[0].second == endAtomNeighbors[1].second);
```

But that's stylistic I think.